### ISSUE TITLE
Predicted Nan labels 

### ISSUE THREAD

**Issue by Mahmedturk** [yandex-research/rtdl#21]
do you have any idea why would the model predict labels as NaN instead of 0 or 1 in a binary classification problem?

**Comment by Yura52** [comment]
@Mahmedturk hi! Sorry for the slow reply. Do you still have this problem?

First, the outputs of most DL models are logits, not labels. You have to convert logits to labels manually in the same manner as described [here](https://github.com/yandex-research/rtdl/issues/8#issuecomment-961445259).

Second, NaNs are not expected even in the case of logits. Usually NaNs indicate that the input contains NaNs, or model's weights contain NaNs, or model's weights are too large in their magnitude (which can be cause by many reasons) etc.

**Comment by Yura52** [comment]
Feel free to reopen the issue if the help is needed!
